content/webapp/services/wellcome/catalogue/types/api.ts

```diff
@@ -1,25 +1,31 @@
-type ItemsLocationsAccessConditionsStatus =
-  | 'open'
-  | 'open-with-advisory'
-  | 'restricted'
-  | 'closed'
-  | 'licensed-resources'
-  | 'unavailable'
-  | 'permission-required'
-  | '!open'
-  | '!open-with-advisory'
-  | '!restricted'
-  | '!closed'
-  | '!licensed-resources'
-  | '!unavailable'
-  | '!permission-required';
+// ItemsLocationsAccessConditionsStatus is unused at the moment as it was very difficult to make it work with the
+// existing encoding/decoding structure. However, it is defined here for future use, and to allow type checking
+// when defining API parameters in other parts of the codebase.
+const allowedStatuses = [
+  'open',
+  'open-with-advisory',
+  'restricted',
+  'closed',
+  'licensed-resources',
+  'unavailable',
+  'permission-required',
+  '!open',
+  '!open-with-advisory',
+  '!restricted',
+  '!closed',
+  '!licensed-resources',
+  '!unavailable',
+  '!permission-required',
+] as const;
+export type ItemsLocationsAccessConditionsStatus =
+  (typeof allowedStatuses)[number];
 
 export type CatalogueWorksApiProps = {
   query?: string;
   page?: number;
   workType?: string[];
   'items.locations.locationType'?: string[];
-  'items.locations.accessConditions.status'?: ItemsLocationsAccessConditionsStatus;
+  'items.locations.accessConditions.status'?: string[];
```

Comment:
I just couldn't understand how to make the encoding and decoding work with a type, and we don't do it for others so simplifying.